packages/core/src/createApp.ts

```diff
@@ -39,6 +45,7 @@ export interface DaVinciOptions {
 			options?: any;
 		};
 	};
+	tls?: DaVinciTlsOptions;
```

Comment:
can we rename the option to `https`?

Additionally, you can probably use the types from https.ServerOptions
```ts
https?: https.ServerOptions;
```

Reply:
Done!